packages/terra-embedded-content-consumer/package.json

```diff
@@ -24,11 +24,12 @@
   "peerDependencies": {
     "react": "^16.8.5",
     "react-dom": "^16.8.5",
-    "xfc": "^1.2.1"
+    "xfc": "git+https://github.com/kolkheang/xfc.git#display-frame-focus-indicator"
   },
   "dependencies": {
     "classnames": "^2.2.5",
-    "prop-types": "^15.5.8"
+    "prop-types": "^15.5.8",
+    "style-to-object": "^0.4.2"
```

Comment:
We should not need to add this third-party dependency.

Reply:
Good call. With the current way, there is just 1 style for focus `outline`, and one style for blur `outline` and we have two variables for these. We can set the value for each into the frame's style directly. Previously, I was thinking that we could set multiple style attributes for focus or blur.